### ISSUE TITLE
Finetune fails when not using BatchNorm

### ISSUE THREAD

**Issue by Lianghy15** [tding1/CDFI#17]
Hi, I notice that your network does not contain BN layers. Is BN really not necessary in your training? When I finetune your pretrained model on my own dataset, the trainging loss doesn' t decrease. But with BN, it decreases normally. So I think maybe with BN or some residual shortcut can improve the model a little?

**Comment by tding1** [comment]
Our network does not contain BN layers does not mean that BN does not have any benefits, as you observed in your own experiments. Feel free to try any new ideas on the network. Also, the network already has residual shortcuts embedded.
